cluster_tolerance: 0.1 # minimal distance to create a cluster
obstacle_movement_tolerance: 1.0 # if obstacle moved this far means it it gone
minimal_survival: 2 # minimal survival counter to confirm a obstacle
maximal_reserve: 10 # maximal reserve counter to confirm obstacle disappear
detect_padding: 0.2
line_segment_distance: 0.1

min_points: 2 # minimal points to confirm a cluster
include_radius: 0.1
exclude_radius: 0.4
distance_filter: 6.0
transfer_distance: 0.2 # transfer obstacle to previous stage
